[2023-06-30T00:00:42.885+0000] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: Oracle_to_Filestore.execute_function_task manual__2023-06-29T23:55:35.526910+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2023-06-30T00:00:42.891+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
